src/web/Dockerfile

```diff
@@ -49,6 +49,12 @@ ENV NEXT_PUBLIC_STATUS_URL ${STATUS_URL}}
 ARG WEB_URL
 ENV NEXT_PUBLIC_WEB_URL ${WEB_URL}
 
+ARG SUPABASE_URL
```

Comment:
@joelazwar, you need to define `SUPABASE_URL` and `ANON_KEY` before `FROM` too so you can use them here.

Reply:
Also, we're missing it in the build, see https://github.com/Seneca-CDOT/telescope/blob/master/src/web/next.config.js#L19-L28

Reply:
Here too yeah